ios-sdk.xcodeproj/project.pbxproj

```diff
@@ -142,29 +140,12 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
-		7D7DAE171E60AFEE00FFCA6F /* resources */ = {
+		7D7847231E6775EA000D56CA /* dimensions */ = {
 			isa = PBXGroup;
 			children = (
-				7D7DAE191E60B00400FFCA6F /* dimens */,
-				7D7DAE181E60AFFE00FFCA6F /* strings */,
+				7D7847241E6775EA000D56CA /* Dimensions.swift */,
```

Comment:
Yes. Its now in `src/dimensions/Dimensions.swift`

I'm thinking we might want to move our controllers, managers, extensions into separate folders in later PR too